react-native/react/tracker/render.desktop.js

```diff
@@ -1,24 +1,50 @@
 'use strict'
+/* @flow */
 
+// $FlowIssue base-react
 import React, {Component} from '../base-react'
 
+// $FlowIssue platform files
 import Header from './header.render'
+// $FlowIssue platform files
 import Action from './action.render'
+// $FlowIssue platform files
 import Bio from './bio.render'
+// $FlowIssue platform files
 import Proofs from './proofs.render'
 
+import type { BioProps } from './bio.render.desktop'
+import type { ActionProps } from './action.render.desktop'
+import type { HeaderProps } from './header.render.desktop'
+import type { ProofsProps } from './proofs.render.desktop'
+
+export type TrackerProps = {
+  bioProps: BioProps,
+  actionProps: ActionProps,
+  headerProps: HeaderProps,
+  proofsProps: ProofsProps
+}
+
 export default class Render extends Component {
-  render () {
+  props: TrackerProps;
+
+  render (): ReactElement {
     return (
       <div style={{display: 'flex', flex: 1, flexDirection: 'column'}}>
-        <Header {...this.props} />
```

Comment:
in reference to the smart component, i think the render should split the props into the sub props and pass those down, cause it does know what needs what